rasa/utils/plotting.py

```diff
@@ -117,103 +117,134 @@ def plot_confusion_matrix(
         fig.savefig(output_file, bbox_inches="tight")
 
 
-def plot_histogram(
-    hist_data: List[List[float]], title: Text, output_file: Optional[Text] = None
-) -> None:
-    """Plots a side-by-side comparative histogram of the confidence distribution.
+def _paired_histogram_specification(
```

Comment:
Can you rename the function to something like `_extract_paired_histogram_specification`. It tells that the function is doing something, i.e. `extract`.